assignment-2/7s.md

```diff
@@ -1,2 +1,296 @@
+##### Skills
+As noticed before, a main skill of KLM is customer engagement through Twitter and Facebook, this helps in creating a very friendly brand name. Other skills of note are the international networking and cooperation through the Skyteam network, ensuring that you can reach virtually any destination of the world while flying with KLM or one of their partners.
+##### Style
+KLM's leadership style is very detached from the 'workfloor'. Captains all have their own style of leadership in their airplane. The main way of influence from higher management is through flight plans and regulations. But as stated earlier, pilots really only see their supervisor once something goes really wrong. So if a captain flies slightly faster than planned, which burns more fuel, to be on their destination earlier that will likely not attract any attention at all.  This means that the area in which leadership has the most influence is in all the supporting acitivities: routes, staff planning, catering, hotels, management, cargo and training.
+##### Shared Values
+The two Shared Values that KLM hold closest to its heart are: the experience of flying and customer service. In their commercials KLM emphasize the first shared value. In their actions on social media and training of flight crew they pay a great deal of attention to the second shared value
+
+## 2.3 Failed Case : HealthCare.gov ##
+
+In the previous sections we've analyzed two successful companies, through the eyes of the 7S model. Both very different organizations with very different perspectives and very different visions and missions. During our previous research we've analyzed the case of HealthCare.gov. This website is an integral part of the in 2010 changed health care system of the United States. President Obama introduced the 'Obamacare'. The HealthCare.gov website must make sure that all American individuals can find and compare health insurance policies via one website. The system was launched on 01 October 2013. It is still live and in that sense it is not a failed case. However, based on our previous study and our analysis we can conclude that the development and organization of the system has been a failure. The original estimation for the costs of the project was $292 million. Before the launch the costs where already $500 million. According to figures from February 2014 the costs exceeded $1.7 billion by then [cite]. Besides the tremendous costs, the launch of the system was surrounded by technical problems. In this section we analyze the HealtCare.gov further, by taking a careful look at the successful companies. Despite the big differences between business, there are organizational practices that may be shared. We want to know what the contractors behind HealtCare.gov could have learned from the practices of Spotify and KLM.
```

Comment:
"where already" => "were"

The commas in the last three sentences are placed kind of oddly. Should probably take a look at them again.